mapie/estimators.py

```diff
@@ -134,7 +134,7 @@ def __init__(
         alpha: float = 0.1,
         method: str = "plus",
         cv: Optional[Union[int, BaseCrossValidator]] = None,
-        n_jobs: int = 1,
+        n_jobs: Optional[Union[int, None]] = None,
```

Comment:
Optional sert justement à prendre en compte les valeurs `None`. Pas besoin d'Union
https://mypy.readthedocs.io/en/stable/kinds_of_types.html#optional-types-and-the-none-type

Reply:
Oui bien vu je me suis compliqué la vie.